### PR TITLE
Custom MAKEFLAGS for Linux builds

### PR DESCRIPTION
**Pull request by jschueller**
supersedes https://github.com/conda-forge/conda-smithy/pull/174

### CODE REVIEW

File: recipe/activate.sh [hunk 1]
```diff
@@ -33,6 +33,7 @@ then
     export CXXFLAGS="${CXXFLAGS} -DBOOST_MATH_DISABLE_FLOAT128"
     export LDFLAGS="${LDFLAGS}"
     export LINKFLAGS="${LDFLAGS}"
+    export MAKEFLAGS="-j2 ${MAKEFLAGS}"
```
Comment (pelson) [inline recipe/activate.sh]: Would you mind adding a comment to help future maintainers understand the rationale?

**Comment by jschueller** [comment]
@pelson, ok added

**Comment by pelson** [comment]
:+1: from me. @jakirkham - over to you.

**Comment by jakirkham** [comment]
So, this should not be in the `toolchain`. This should live somewhere where we configure CIs, but not here. Perhaps as part of this ( https://github.com/conda-forge/conda-forge-build-setup-feedstock ) or some other yet to be created package. Keep in mind that `MAKEFLAGS` is automatically passed through.

**Comment by jschueller** [comment]
@jakirkham, ok I reopened https://github.com/conda-forge/conda-smithy/issues/174

**Comment by jakirkham** [comment]
Thanks @jschueller. Though I imagine we will still want to move it somewhere else, it makes sense to track it on conda-smithy until we figure that out.

**Comment by jschueller** [comment]
@jakirkham ok see https://github.com/conda-forge/conda-forge-build-setup-feedstock/pull/1

**Comment by jakirkham** [comment]
Awesome. Thanks.

**Comment by pelson** [comment]
Sorry for sending you on a wild-goose chase @jschueller.
Thanks for correcting the course @jakirkham. I agree.

**Pull request closed without merging**